validation-test/Sema/SwiftUI/issue-59716.swift

```diff
@@ -9,8 +9,8 @@ extension TestLayout {
     return f()
   }
 }
-struct EqualWitdthHStack : TestLayout {}
-extension EqualWitdthHStack: View {
+struct EqualWidthHStack : TestLayout {}
```

Comment:
Again, these types of changes should be avoided IMO. In this specific case, this is a test program so it doesn't matter, but this runs the risk of an ABI break.